sycl/test/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ list(APPEND SYCL_DEPLOY_TEST_DEPS
 add_lit_testsuite(check-sycl-deploy "Running the SYCL regression tests"
   ${CMAKE_CURRENT_BINARY_DIR}
   ARGS ${DEPLOY_RT_TEST_ARGS}
-  PARAMS "SYCL_BE=PI_OPENCL"
+  PARAMS "SYCL_BE=opencl"
```

Comment:
I think it would be nice to rename `SYCL_BE` LIT config parameter to avoid confusion with environment variable.
E.g. `SYCL_PLUGIN`.